src/classes/Lookups.ts

```diff
@@ -0,0 +1,172 @@
+import { FormTypes } from '@oneblink/types'
+import OneBlinkAPI from '../lib/one-blink-api'
+import { ConstructorOptions } from '../types'
+
+export default class Lookups extends OneBlinkAPI {
+  /**
+   * #### Example
+   *
+   * ```typescript
+   * import { Lookups } from '@oneblink/sdk'
+   *
+   * const options = {
+   *   accessKey: '123455678901ABCDEFGHIJKL',
+   *   secretKey: '123455678901ABCDEFGHIJKL123455678901ABCDEFGHIJKL',
+   * }
+   * const lookups = new Lookups(options)
+   * ```
+   */
+  constructor(options: ConstructorOptions) {
+    options = options || {}
+    super(options.accessKey, options.secretKey)
+  }
+
+  /**
+   * #### Example
+   *
+   * ```javascript
+   * const lookupId = 1
+   * const lookup = await lookups.getLookup(lookupId)
+   * // Use lookup here
+   * ```
+   *
+   * @param id The exact id of the lookup you wish to get
+   * @returns
+   */
+  getLookup(id: number): Promise<FormTypes.FormElementLookup> {
+    if (typeof id !== 'number') {
+      return Promise.reject(new TypeError('Must supply "id" as a number'))
+    }
+
+    return super.getRequest(`/form-element-lookups/${id}`)
+  }
+
+  /**
+   * #### Example
+   *
+   * ```javascript
+   * const organisationId = '2i4321a7n2389a2700065425'
+   * const lookups = await lookups.searchLookups({
+   *   organisationId,
+   *   limit: 50,
+   *   offset: 0,
+   * })
+   * // Use lookups here
+   * ```
+   *
+   * @param options
+   * @returns
+   */
+  searchLookups(options: {
+    /** The exact id of the lookup you wish to get */
+    organisationId: string
```

Comment:
No need to pass this, the key is scoped to the organisation.

Reply:
Endpoint requires it

Reply:
we should update the endpoint.

Reply:
We discussed yesterday and decided not to for now. We can discuss it again if you'd like?